configuration/scripts/options/set_nml.jra55_2008

```diff
@@ -9,7 +9,7 @@ maskhalo_dyn   = .true.
 maskhalo_remap = .true.
 maskhalo_bound = .true.
 fyear_init      = 2008
-atm_data_dir    = 'ICE_MACHINE_INPUTDATA/CICE_data/forcing/gx1/JRA55'
+atm_data_dir    = '/ICE_MACHINE_INPUTDATA/CICE_data/forcing/gx1/JRA55'
```

Comment:
Looks like there is a typo here (extra slash in front).

Reply:
I did not make any changes to the wave implementation code. It is possible I pulled from master. Can you put the jra55 forcing data on gordon at /p/work1/RASM_data etc.?
Then I can re-run the test suites